package.json

```diff
@@ -14,7 +14,7 @@
     "bumpManifestVer": "node ./bin/bumpManifestVer.js"
   },
   "dependencies": {
-    "@deskpro/app-sdk": "2.1.14",
+    "@deskpro/app-sdk": "2.1.12",
```

Comment:
Why was the SDK downgraded?

Reply:
because `iframe-resizer` not released yet